js/src/admin.js

```diff
@@ -2,6 +2,67 @@
  * @package Polylang
  */
 
+window.pll = window.pll || {};
```

Comment:
This is an old fashion way of accessing `pll` from other script sheets. This is not what we want here.
Also it would break without error if some third party do the same.
Please use `const pllMachineTranslationButton` or something instead.

Reply:
It wouldn't break unless the plugin does this directly:
```JS
window.pll = { ... };
```
In this case we can say it's intentional.

`const` added in [411277a](https://github.com/polylang/polylang/pull/1420/commits/411277a5b02769446f93f41ed8cbed2522dddc41).